frontend/src/containers/OnBoarding.js

```diff
@@ -99,18 +105,22 @@ export class OnBoarding extends Component {
 
     return validateSchema(githubForm.attributes, githubSchema)
       .then(() => createGroupFromGithubRepo(payload, attr.token))
-      .then(() => this.setState({step: 5}))
+      .then(() => this.setState({step: 4}))
```

Comment:
Same infinite loop comment here. We don't support back in our flow... maybe worth thinking about?